src/ott/solvers/linear/continuous_barycenter.py

```diff
@@ -31,21 +31,24 @@ class FreeBarycenterState(NamedTuple):
   """Holds the state of the Wasserstein barycenter solver.
```

Comment:
Consider refactoring this also as a dataclass.

Reply:
same comment, our states are also `NamedTuple`.